syz-manager/manager.go

```diff
@@ -182,12 +184,18 @@ func RunManager(cfg *mgrconfig.Config, target *prog.Target, sysTarget *targets.T
 		reproRequest:          make(chan chan map[string]bool),
 		usedFiles:             make(map[string]time.Time),
 		saturatedCalls:        make(map[string]bool),
+		kcovFilter:            &CoverFilter{enableFilter: false},
 	}
 
 	mgr.preloadCorpus()
 	mgr.initHTTP() // Creates HTTP server.
 	mgr.collectUsedFiles()
 
+	mgr.initKcovFilter()
+	if mgr.kcovFilter.enableFilter {
+		mgr.kcovFilter.createBitmap()
```

Comment:
This should a part of initKcovFilter. It is logically is. No point in splitting init in 2 parts and bothering this higher-level function what that detail.

Looking further, I think the interface should be just:
```
func createCoverageFilter(cfg *mgrconfig.Config) (coverageFilterFilename string, err error)
```
Then, Manager should hold just the filename. If it's empty, filter is not enabled.

This seems to be what the actual essence of the contract.

We can have coverageFilter type as a helper to implement createCoverageFilter, if it makes things simpler and better organized.